Cabal/doc/getting-started.rst

```diff
@@ -0,0 +1,159 @@
+Getting Started with Haskell and Cabal
+======================================
+
+Installing the Haskell toolchain
+--------------------------------
+
+To install the Haskell toolchain follow the `ghcup instructions
+<https://www.haskell.org/ghcup/>`__ if you're using Linux or Mac, or follow
+`this guide <https://hub.zhox.com/posts/introducing-haskell-dev/>`__ if you're
+using Windows.
+
+
+Creating a new application
+--------------------------
+
+Let's start by creating a simple Haskell application from scratch where we'll
+learn about a Haskell package's directory structure, how to run the executable,
+and how to add external dependencies.
+
+
+Initializing the application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by creating a ``myfirstapp`` directory to hold the project files:
+
+.. code-block:: console
+
+    $ mkdir myfirstapp && cd myfirstapp
```

Comment:
in windows powershell (present by default in last windows versions) could be
```
>mkdir myfirstapp ; cd myfirstapp
```
in raw dos
```
> md myfirstapp && cd myfirstapp
```

Reply:
```
mkdir myfirstapp ; cd myfirstapp
```

would work in unix shells to, but I think it's better to split into two separate commands

Reply:
Split into two separate commands.